### ISSUE TITLE
Load pages from 51 on Sankaku

### ISSUE THREAD

**Issue by Ma0ux** [Nandaka/DanbooruDownloader#162]
I've just see that you provided a fix fro issue 111, but it hasn't worked, since I can't load the page 51. Maybe I am doing something wrong. Can you tell me how can I use the fix? How can I update? Please?

**Comment by Nandaka** [comment]
Can you open the website from browser? Ensure you supply the login/cookie value for sankaku, or they limit the page for searching.

From readme:
```
Q7: How to login to Gelbooru/Sankaku/get cookie value?
A7: Follow this step:
    1. Press F12 on your Chrome browser and select Network tab.
    2. Go to the booru site and login.
    3. Click one of the entry and copy the Cookie value from the Request Header. 
       For gelbooru, it should like this: user_id=<number>; pass_hash=<long string>
       For sankaku, login=<username>; pass_hash=<long string>;
    4. Paste the Cookie value to the Username field.
    5. Set Login Type to Cookie. Refer to http://i.imgur.com/rCCjnPs.png
```

![image](https://user-images.githubusercontent.com/836867/37238802-f28df08a-23e2-11e8-991e-6ff45fe0a0d6.png)

**Comment by Ma0ux** [comment]
Thank you for the answer.

I did that, the login. 
![this2](https://user-images.githubusercontent.com/37232136/37246362-605c6376-247e-11e8-8e3e-800cb0f08fdf.jpg)
I got this when I tried to get into page 51, with the "tags=vocaloid&page=51&next=" on the url bar.
![this1](https://user-images.githubusercontent.com/37232136/37246384-a45a25ea-247e-11e8-8dc9-d5d7b3d966f3.jpg)
But when I use the "next"  button on the page, I got this:
![this4](https://user-images.githubusercontent.com/37232136/37246399-d3f5638c-247e-11e8-8bfb-6f32a46c5025.jpg)
And the url there is different. This is what happens when I try to get into page 51 with danbooru:
![this5](https://user-images.githubusercontent.com/37232136/37246417-f4eaae26-247e-11e8-868f-78a112ae8e9f.jpg)
And the login is set, as you can see in the first image. What am I doing wrong?

Thank you again.

**Comment by Nandaka** [comment]
Try to change QueryStringHtml to `/?%_query%`?

![image](https://user-images.githubusercontent.com/836867/37247216-df32a06e-246c-11e8-82b7-b1878c82a504.png)

**Comment by Ma0ux** [comment]
Nope. I still get error 404. 
could it be a matter of my Sankaku account?

**Comment by Nandaka** [comment]
Maybe? My level is Contributor.
![image](https://user-images.githubusercontent.com/836867/37247689-f8f9aad6-2473-11e8-8cab-b792e6095412.png)

**Comment by Nandaka** [comment]
Try this way? If it is working, then maybe I can set a workaround.
![image](https://user-images.githubusercontent.com/836867/37247704-52e20a02-2474-11e8-90c7-248dce3f274c.png)

**Comment by Ma0ux** [comment]
Awesome. It kind of worked, but it's showing the page 52.
![this6](https://user-images.githubusercontent.com/37232136/37247913-21dc13be-249a-11e8-80e8-50ef692ddc7b.jpg)
What does it mean? It can be worked somehow? That would be great.

**Comment by Nandaka** [comment]
Yes, but with some limitation, as I need to get the post id from the previous page to make it works. so If you try to jump to page 100, I need to get the last image in page 99, which in turn I need to fetch the previous image from page 98 and so on until it hit the page 50 (as this is the limit for normal user).
This is not effective, unless you are ok to key in the value of `?next=<first_image_in_target_page>` every time you query sankaku with page > 50.

Btw, anon can only browse to page 25 now:
![image](https://user-images.githubusercontent.com/836867/37247953-76db9284-2479-11e8-9018-f2aa954d9a7a.png)

**Comment by Ma0ux** [comment]
Yeah. Maybe it's a matter of the level of your account. How did you get that level? And the other guys have the same level? To try and see what can I do.

**Comment by Nandaka** [comment]
I think I get it by updating the tagging information in the site? E.g. update the artist url in the tags details.

**Comment by GiRaFa-SM** [comment]
Ye, but by doing this it'll only load the page you're and wont update past to this, tested this in the last released ver.

**Comment by GiRaFa-SM** [comment]
found an workaround till dev fix this.

Load page 49 > scroll till page 51 loads > after that type page number you stopped (101 for example), it'll load starting from the page selected.
